source/features/sticky-conversation-sidebar.tsx

```diff
@@ -6,11 +6,15 @@ import * as pageDetect from 'github-url-detection';
 import features from '.';
 import onReplacedElement from '../helpers/on-replaced-element';
 
-const sideBarSelector = '#partial-discussion-sidebar, .discussion-sidebar';
+const sideBarSelector = [
+	'#partial-discussion-sidebar, .discussion-sidebar', // Conversations
+	'.repository-content .flex-column > :last-child [data-pjax]' // `isRepoRoot`
+].join();
 
 function updateStickiness(): void {
 	const sidebar = select(sideBarSelector)!;
-	const sidebarHeight = sidebar.offsetHeight + 60; // 60 matches sticky header's height
+	const stickyHeight = (pageDetect.isIssue() || pageDetect.isPRConversation()) ? 60 : 0;
+	const sidebarHeight = sidebar.offsetHeight + stickyHeight;
```

Comment:
Don't strip comments

```suggestion
	const margin = pageDetect.isIssue() || pageDetect.isPRConversation() ? 60 : 0; // 60 matches sticky header's height
	const sidebarHeight = sidebar.offsetHeight + margin;
```

Reply:
oops, I moved that variable too much and lost it in the process :pray: